homeassistant/components/zone/trigger.py

```diff
@@ -106,7 +103,7 @@ def zone_automation_listener(zone_event):
         ):
             description = f"{entity} {_EVENT_DESCRIPTION[event]} {zone_state.attributes[ATTR_FRIENDLY_NAME]}"
             hass.async_run_hass_job(
-                job,
+                job,  # type: ignore[arg-type]
```

Comment:
I think this is an issue in `mypy`

`Argument 1 to "async_run_hass_job" of "HomeAssistant" has incompatible type "HassJob[[Dict[str, Any], Optional[Context]], Coroutine[Any, Any, None]]"; expected "HassJob[[VarArg(Any), KwArg(Any)], Coroutine[Any, Any, None]]"`